reference/api/rest-api/endpoints/team-collaboration-endpoints/rooms-endpoints/export-room.md

```diff
@@ -20,7 +20,7 @@ This requires the user to have the `mail-messages` permission.
 | `type`     | `email` or `file`    | Required | How you want the room to be exported. |
 | `dateFrom` | `2020-01-13`         | Optional | Startdate to begin fetching.          |
 | `dateTo`   | `2021-12-13`         | Optional | End date for fetching.                |
-| `format`   | `html` or `json`     | Optional | The file type to export as.           |
+| `format`   | `html` or `json`     | Required | The file type to export as.           |
```

Comment:
Correct. It's required to set the export `format` when the export type is file. Otherwise you get an error:
``` json
{
    "success": false,
    "error": "[error-invalid-format]",
    "errorType": "error-invalid-format"
}
```